_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/templates/video-resource/node--video-resource.html.twig

```diff
@@ -14,7 +14,7 @@
                 {% endif %}
             </li>
             <li>
-                <small>Published: {{ date }}</small>
+                <small>Published: {{ node.createdtime|date('D, m/d/Y') }}</small>
```

Comment:
There's also a field_video_published_date (I think that's the correct name) you could use. I think we're going to start using that for other things, probably should use it here.